maxim/tests/mock_writer.py

```
@@ -0,0 +1,260 @@
+
+    def assert_entity_count(self, entity, expected_count):
+        """Assert the number of logs for a specific entity."""
+        logs = self.get_logs_by_entity(entity)
+        actual_count = len(logs)
+        assert (
+            actual_count == expected_count
+        ), f"Expected {expected_count} '{entity}' logs, got {actual_count}"
+
+    def print_logs_summary(self):
+        """Print a summary of all captured logs for debugging."""
+        print(f"\n=== Mock Writer Log Summary ===")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)